File: exercises/beecrowd/1011/1011.go
// https://judge.beecrowd.com/pt/problems/view/1011
package main

import (
	"fmt"
	"math"
)

func main() {
	const PI float64 = 3.14159
	var r float64

	fmt.Scanln(&r)

	total := (4.0 / 3.0) * PI * math.Pow(r, 3)

	fmt.Printf("VOLUME = %.3f\n", total)
}

File: exercises/beecrowd/1011/1011.ts
// https://judge.beecrowd.com/pt/problems/view/1011
import * as fs from 'fs';

function main(): void {
    const input = fs.readFileSync('/dev/stdin', 'utf8');
    const lines = input.split(/\r?\n/);
    
    const PI = 3.14159
    const r = Number(lines.shift());
    
    const result = ((4.0 / 3.0) * PI * Math.pow(r, 3)).toFixed(3)
    
    console.log(`VOLUME = ${result}`);
}

main()